tests/foreman/ui_airgun/test_package.py

```diff
@@ -0,0 +1,250 @@
+        session, module_org, module_yum_repo, module_yum_repo2):
+    """Create product with two different yum repositories assigned to it.
+    Search for packages inside of these repositories. Make sure that unique
+    packages present in corresponding repos.
+
+    :id: 249ac04b-8e31-42e9-ac37-08608bf867a1
+
+    :expectedresults: Content search functionality works as intended and
+        expected packages are present inside of repositories
+
+    :BZ: 1514457
+
+    :CaseLevel: Integration
+    """
+    with session:
+        session.organization.select(org_name=module_org.name)
+        assert session.package.search(
+            'name = tiger')[0]['RPM'].startswith('tiger')
+        assert session.package.search(
+            'name = Lizard')[0]['RPM'].startswith('Lizard')
+        # First repository
+        assert session.package.search(
+            'name = tiger', repository=module_yum_repo.name
+        )[0]['RPM'].startswith('tiger')
+        assert not session.package.search(
+            'name = Lizard', repository=module_yum_repo.name
+        )
+        # Second repository
+        assert session.package.search(
+            'name = Lizard', repository=module_yum_repo2.name
+        )[0]['RPM'].startswith('Lizard')
+        assert not session.package.search(
+            'name = tiger', repository=module_yum_repo2.name
+        )
+
+
+@tier2
+@upgrade
+def test_positive_check_package_details(session, module_org, module_yum_repo):
+    """Create product with yum repository assigned to it. Search for
+    package inside of it and then open it. Check all the details about that
+    package
+
+    :id: 57625386-4a9e-4bea-b2d5-d97326043150
+
+    :expectedresults: Package is present inside of repository and has all
+        expected values in details section
+
+    :CaseLevel: Integration
+    """
+    with session:
+        session.organization.select(org_name=module_org.name)
+        expected_package_details = {
+            'description': 'A dummy package of gorilla',
+            'summary': 'A dummy package of gorilla',
+            'group': 'Internet/Applications',
+            'license': 'GPLv2',
+            'url': 'http://tstrachota.fedorapeople.org',
+            'size': '2.39 KB (2452 Bytes)',
+            'filename': 'gorilla-0.62-1.noarch.rpm',
+            'checksum': ('ffd511be32adbf91fa0b3f54f23cd1c02add50578344'
+                         'ff8de44cea4f4ab5aa37'),
+            'checksum_type': 'sha256',
+            'source_rpm': 'gorilla-0.62-1.src.rpm',
+            'build_Host': 'smqe-ws15',
+            'build_time': '1331831364',
+        }
+        all_package_details = session.package.read(
+            'gorilla', repository=module_yum_repo.name)['details']
+        package_details = {
+            key: value for key, value in all_package_details.items()
+            if key in expected_package_details
+        }
+        assert expected_package_details, package_details
```

Comment:
isn't that a typo? did you mean `assert expected_package_details == package_details`? i'm afraid your assertion will always pass